react-web/src/components/molecules/input-text/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+
+import classNames from 'classnames'
+
+import { IInputProps } from 'components/molecules/input-text/type'
+
+import Eth from 'app/core/resources/eth-icon.svg'
+
+import { Label } from '../../atoms/typography/label'
+import styles from './styles.module.scss'
+
+export enum InputLabel {
+  sending = 'Sending',
+  receive = 'Receive',
+}
+
+export enum CurrencyLabel {
+  eth = 'ETH',
+  weth = 'WETH',
+}
```

Comment:
I don't have much context of this project just yet, so I am just a little bit confused 😅 

Is it okay to have specific information on this `input-text` molecule? I say this because I noticed it has things like this enum above and balance info below, and I was expecting to see a standardized component here because of the name of the file 🤔

Reply:
we can change the name to Labeled Input, Transaction Input or something like that, since we will have a single screen and a single type of Input.
What do you think? 🤔